guestbook/README.md

```diff
@@ -4,704 +4,294 @@
+        kubectl get pods
 
-```console
-$ kubectl get deployments
-NAME           DESIRED   CURRENT   UP-TO-DATE   AVAILABLE   AGE
-redis-master   1         1         1            1           27s
-```
+   The response should be similar to this:
 
-Then, you can list the pods in the cluster, to verify that the master is running:
+        NAME                            READY     STATUS              RESTARTS   AGE
+        redis-master-1068406935-3lswp   1/1       Running             0          1m
+        redis-slave-2005841000-fpvqc    0/1       ContainerCreating   0          6s
+        redis-slave-2005841000-phfv9    0/1       ContainerCreating   0          6s
+        
+### Creating the Redis Slave Service
+The guestbook application needs to communicate to Redis workers to read data. To make the Redis workers discoverable, you need to set up a Service. A Service provides transparent load balancing to a set of Pods.
 
-```console
-$ kubectl get pods
-```
+1. Apply the Redis Slave Service from the following `redis-slave-service.yaml` file:
 
-You'll see all pods in the cluster, including the redis master pod, and the status of each pod.
-The name of the redis master will look similar to that in the following list:
+        kubectl apply -f redis-slave-service.yaml
 
-```console
-NAME                            READY     STATUS    RESTARTS   AGE
-redis-master-2353460263-1ecey   1/1       Running   0          1m
-...
-```
+include code.html language="yaml" file="redis-slave-service.yaml" ghlink="/docs/tutorials/docs/tutorials/stateless-application/redis-slave-service.yaml"
 
-(Note that an initial `docker pull` to grab a container image may take a few minutes, depending on network conditions. A pod will be reported as `Pending` while its image is being downloaded.)
+{:start="2"}
+2. Query the list of Services to verify that the Redis Slave Service is running:
 
-`kubectl get pods` will show only the pods in the default [namespace](http://kubernetes.io/docs/user-guide/namespaces/).  To see pods in all namespaces, run:
+        kubectl get services
 
-```
-kubectl get pods --all-namespaces
-```
+   The response should be similar to this:
 
-For more details, please see [Configuring containers](http://kubernetes.io/docs/user-guide/configuring-containers/) and [Deploying applications](http://kubernetes.io/docs/user-guide/deploying-applications/).
+        NAME           CLUSTER-IP   EXTERNAL-IP   PORT(S)    AGE
+        kubernetes     10.0.0.1     <none>        443/TCP    2m
+        redis-master   10.0.0.151   <none>        6379/TCP   1m
+        redis-slave    10.0.0.223   <none>        6379/TCP   6s
 
-#### Optional Interlude
+## Set up and Expose the Guestbook Frontend
+This tutorial uses a simple PHP server that is configured to talk to either the slave or master Services, depending on whether the client request is a read or a write. It exposes a simple JSON interface, and serves a jQuery-Ajax-based UX.
```

Comment:
This is a bit too much detailed for anyone following the tutorial.  suggested rewrite:

> The guestbook application has a web frontend serving the HTTP requests written in PHP. It is configured to connect to the `redis-master` service for write requests and the `redis-slave` service for read requests.